assignment_1/src/State.java

```diff
@@ -67,9 +67,68 @@ public String toString(){
 	public boolean isLeaf() {
 		return true;
 	}
-
+    /*
+        Check the values of the current position of the given agent and the values of the positions around the agent and returns the possible legal moves.
+    */
 	public Vector<String> legalMoves(int agent) {
-		return new Vector<>();
+        Vector<String> possible_moves = new Vector<String>();
```

Comment:
You can remove the "String" the new vector like this.
![image](https://user-images.githubusercontent.com/34271434/192161996-b894b8e0-9f75-48e4-aed9-80b865dd26a8.png)